site/docs/v1/tech/oauth/integrations/salesforce.adoc

```diff
@@ -0,0 +1,213 @@
+---
+layout: doc
+title: "OAuth Integrations: Salesforce"
+description: A practical approach of using FusionAuth SSO capabilities as a login provider in a Salesforce application
+navcategory: login-methods
+---
+
+== Overview
+
+link:https://www.salesforce.com/[Salesforce] is a robust customer relationship management software that allows users to sign in into their accounts using an external Identity Provider. This document covers the configuration necessary to get Salesforce working with FusionAuth as the identity provider using OpenID Connect.
+
+include::docs/v1/tech/samlv2/_saml_idp_note.adoc[]
```

Comment:
I used it because we have this kind of intro in [Sendgrid](https://fusionauth.io/docs/v1/tech/samlv2/sendgrid), [Zendesk](https://fusionauth.io/docs/v1/tech/samlv2/zendesk) and other articles

Reply:
@mooreds any thoughts on this?